src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -9,36 +9,42 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
 import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.common.collect.ImmutableList;
 import com.google.edith.servlets.Item;
-import com.google.edith.servlets.UserInsights;
+import com.google.edith.servlets.UserInsightsService;
+import com.google.edith.servlets.UserInsightsInterface;
+import com.google.edith.servlets.WeekInfo;
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import org.junit.After;
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
-
 
 public final class UserInsightsTest {
```

Comment:
Usually, the test file should be named the same as the file it is testing. In this case, in this case `UserInsightsServiceTest` would be better.

Reply:
Okay, I changed it.